Core/clim-basic/drawing/medium.lisp

```diff
@@ -295,8 +300,9 @@
 (defun parse-text-style (style)
   (cond ((text-style-p style) style)
         ((null style) (make-text-style nil nil nil)) ; ?
-        ((and (listp style) (<= 3 (length style) 4))
-         (apply #'make-text-style style))
+        ((and (listp style) (alexandria:length= 3 style))
+         (destructuring-bind (family face size) style
+           (make-text-style family face size)))
```

Comment:
I think this could be a separate commit which should also add tests for `parse-text-style`.

Reply:
fixed